### ISSUE TITLE
NullDc cannot find plugin?  I checked, and it's definitely there.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [skmp/nulldc#128]
```
What steps will reproduce the problem?

1: Run Nulldc



What is the expected output? What do you see instead?

The expected output is: A startup screen with no errors

But instead I see: A dialog box stating "Unable to load nullPvr_Win32.dll 
plugin, errorlevel=-104"



What version of the product are you using? What build? What plugins?

I use the build from the download tab from this website, just like everyone 
else.

I use the following plugins:

PowerVR Plugin: ORIGINAL
GDRom Plugin: ORIGINAL
AICA Plugin: ORIGINAL
ARM7 Plugin: ORIGINAL
Maple Plugin(s): ORIGINAL
Ext.Device Plugin: ORIGINAL



On what kind of system?

My system specifications are as follows:

Operating System: Windows XP Home Edition SP3 x86
CPU: AMD Athlon 2400+ (2.00Ghz single-core)
Video Card: Nvidia GeForce 4 MX Integrated GPU with 64 MB video RAM
Sound Card: Realtek AC'97
Additional related hardware and/or software: 512MB RAM, all drivers are most 
recent official versions, proper prerequisite software was installed 
successfully.



Please provide any additional information below:

I will explain my EXACT steps:
I downloaded nullDC v1.0.4 r22, extracted with WinRAR, added the necessary BIOS 
files to the Data folder, named them properly (dc_boot, dc_flash), and ran 
nullDC_Win32_Release-NoTrace.  I saw a nice main screen, then an error with the 
message "Unable to load nullPvr_Win32.dll plugin, errorlevel=-104"  I am then 
automatically brought to the plugin selection screen (probably because it 
"couldn't find a plugin" and it needs me to specify a new one.  The only 
PowerVR (graphics) plugin I can choose from is "nullDC Power VR -- Direct3D HAL 
[May 13 2010] (drkPvr_Win32.dll)"  Naturally, I choose that, and I don't touch 
any other plugins.  I click OK, and I get a "nullDC_Win32_Release-NoTrace.exe 
has encountered a problem and needs to close.  We are sorry for the 
inconvenience." error, which is the typical windows error, right?  I continue 
with the error message, nullDC closes out, and I am bummed out and confused.  I 
try again, only this time, I don't even get the "couldn't find plugin" error, 
it just skips to the "nullDC_Win32_Release-NoTrace.exe has encountered a 
problem and needs to close.  We are sorry for the inconvenience."  If I deleted 
the folder and re-extract, I can at least start over and get the original bunch 
of errors and I can get to the plugin select screen.  If I hit cancel at the 
plugin select screen, it says "Unable to load plugins -- exiting".  I hit OK 
and the program closes.  If I run it again, it gives me the original bunch of 
messages, as if nothing happened.  This is as far as I can get.

In short, nullDC can't seem to find a plugin, and when a new one is specified, 
it crashes.  If I run again, it crashes immediately, and will continue to do so 
unless I delete and re-extract.  If I hit cancel at the plugin select screen, 
it exits, but I can keep trying, and it wont' crash instantly, but i'm put 
right back at square one.

Please try to fix this.  I've tried so many combinations, so many "fixes", but 
with no luck.  I also spent a lot of time typing this up, so I don't want all 
that work to go to waste.

Don't be afraid to use geeky technical talk, I myself am a computer genius, and 
I can understand what you're saying.


```

Original issue reported on code.google.com by `hookonso...@gmail.com` on 11 Jun 2010 at 10:34
- Merged into: #4
